CMakeLists.txt

```diff
@@ -309,6 +309,14 @@ include_directories(SYSTEM ${LIBUV_INCLUDE_DIRS})
 find_package(Msgpack 1.0.0 REQUIRED)
 include_directories(SYSTEM ${MSGPACK_INCLUDE_DIRS})
 
+option(PREFER_LUAJIT "Prefer LuaJIT over Lua when compiling executable. Test library always uses luajit." ON)
+
+find_package(LuaJit REQUIRED)
```

Comment:
This seems to be what's causing our [nightly builds] to fail.

[nightly builds]: https://code.launchpad.net/~neovim-ppa/+recipe/neovim-unstable